skill-tree/src/test/java/com/RDS/skilltree/EndorsementsIntegrationTests.java

```diff
@@ -37,7 +37,7 @@ public EndorsementsIntegrationTests(UserService userService, UserRepository user
     }
 
     @BeforeEach
-    private void addData() throws MalformedURLException {
+    public void addData() throws MalformedURLException {
```

Comment:
private to public, is this really needed, please revert to private

Reply:
I had changed it since I got a prompt that @BeforeEach and @AfterEach functions should be public, will revert it